modules.d/01fips-crypto-policies/module-setup.sh

```diff
@@ -11,6 +11,7 @@ check() {
 
 # called by dracut
 depends() {
+    echo fips
```

Comment:
In general, this looks good to me.

There's one pitfall, that we should consider but probably ignore as a misconfiguration:
- users with `fips=1` on the kernel command line, and
- `/boot` on a separate filesystem, and
- no `boot=UUID=$fs_uuid` parameter, and
- the `fips` dracut module not enabled

Those configurations would start to fail to boot when the `fips` module is enabled. Personally, I believe this is a misconfiguration, since it skips the kernel integrity checks that would be required by FIPS, so it's better to make this explicitly fail rather than continue to silently ignore it.